src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMethodSymbol.vb

```diff
@@ -1732,8 +1732,6 @@ lReportErrorOnTwoTokens:
                         Return
                     End If
                 End If
-
-                MyBase.DecodeWellKnownAttribute(arguments)
```

Comment:
>MyBase.DecodeWellKnownAttribute(arguments) [](start = 16, length = 42)

Why this call is removed?

Reply:
Take a look at `SourceMethodSymbol.DecodeWellKnownAttribute`. It calls `DecodeWellKnownAttributeAppliedToMethod` and just after that it makes a second call to `MyBase...`. Since this is the only reference to `DecodeWellKnownAttributeAppliedToMethod`, we're unnecessarily calling `MyBase...` again. I was able to detect it because the diagnostics for SkipLocalsInitAttribute in methods and accessors were being reported twice.